tools/coqdep.ml

```diff
@@ -444,6 +444,7 @@ let usage () =
   eprintf "  -boot : For coq developers, prints dependencies over coq library files (omitted by default).\n";
   eprintf "  -sort : output the given file name ordered by dependencies\n";
   eprintf "  -noglob | -no-glob : \n";
+  eprintf "  -f file : use filenames found in the _CoqProject file FILE .v files only, options and other files are ignored)";
```

Comment:
I think it would be more polite to give an error, rather than ignore (is it true that they are ignore BTW)?

Also, the _CoqProject file contains -I and -R directives... if this is the new model I think that all of these should be read from the _CoqProject file, not just the list of files and CoqMakefile.in adapted accordingly.

Reply:
>I think it would be more polite to give an error, rather than ignore

I mean it ignores the other options and non-vfile files found in the coqproject, not the other options/files on the command line. So there is no error to give.

Reply:
Reading options from the coqproject brings up the same issues with passing options part in the coqproject and part in the command line as for vfiles, I don't want to deal with that.

Reply:
I see it is more work, but having the two tools share the argument parsing / proj file format is desirable to me.  Before they were completely different. With this patch they are very close but not equal. bleah...